src/main/java/com/faforever/moderatorclient/mapstruct/GameMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.faforever.moderatorclient.mapstruct;
+
+import com.faforever.moderatorclient.api.dto.Game;
+import com.faforever.moderatorclient.ui.domain.GameFX;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+
+import java.util.List;
+
+@Mapper(uses = {JavaFXMapper.class, PlayerMapper.class, FeaturedModMapper.class, MapVersionMapper.class})
+public abstract class GameMapper {
+    @Mapping(target = "reviews", ignore = true)
+    public abstract GameFX map(Game game);
```

Comment:
Backreferences for GamePlayerStatsFX are missing (GamePlayerStatsFX usage see later comment)

Reply:
oh but it did not make any problems hmm